Entities/Items/Projectiles/Arrow.as

```diff
@@ -155,6 +155,24 @@ void onTick(CBlob@ this)
 
 			processSticking = false;
 		}
+		
+		if (arrowType == ArrowType::bomb)
+		{
+			if (!this.exists("old velocity") || this.get_f32("old velocity") > shape.vellen)
```

Comment:
would it not be easier to compare this.getVelocity and this.getOldVelocity

Reply:
(also of note, there's also oldvellen var, but its not been exposed to angelscript, could be worth doing that at some point)